docs/api-docs/apps/guide/apps-06-callbacks.md

```diff
@@ -1,38 +1,51 @@
-# Single-Click App Callbacks
+# Handling Single-Click App Callbacks
 
 
 <div class="otp" id="no-index">
 
 ### On this page
 - [Overview](#overview)
-- [Load callback](#load-callback)
-- [Uninstall callback](#uninstall-callback)
-- [Remove user callback](#remove-user-callback)
-- [Verifying the signed payload](#verifying-the-signed-payload)
-- [Identifying users](#identifying-users)
+- [Opening app settings: the `/load` callback](#opening-app-settings-the-load-callback)
+- [Removing the app: the `/uninstall` callback](#removing-the-app-the-uninstall-callback)
+- [Removing a user: the `/remove_user` callback](#removing-a-user-the-remove_user-callback)
+- [Verifying the payload](#verifying-the-payload)
+- [Working with payload claims](#working-with-payload-claims)
 - [Code samples](#code-samples)
 - [Helpful tools](#helpful-tools)
 - [Next steps](#next-steps)
 - [Resources](#resources)
 
 </div>
 
-After installing a single-click app, store owners and authorized users *load* the app by clicking the app's icon in the control panel. Store owners can also *uninstall* your app and *remove users* they've authorized to use it. Each of these events triggers a `GET` request (or "callback") from BigCommerce to your app's callback URL configured in the [Developer Portal](https://devtools.bigcommerce.com/my/apps). This article describes how your app should handle each callback and explains how to [verify the `signed_payload`](#verifying-the-signed-payload) and [identify users](#identifying-users).
+After a store owner installs your single-click app, they and their authorized users will need to manage it and configure any settings. Because your app's users will be working in the BigCommerce control panel, certain user-initiated events that might be important to your internal logic will prompt our servers to send your app `GET` requests at predictable endpoints. Each of these requests will return a JSON web token, or _JWT_, as the value of the `signed_payload_jwt` parameter.  This article is both a reference and a guide for developing callbacks to handle these events and authenticate our `GET` requests to these endpoints.

```

Comment:
```suggestion
After a store owner installs your single-click app, they and their authorized users will need to manage it and configure any settings. Because your app's users will be working in the BigCommerce control panel, certain user-initiated events that might be important to your internal logic will prompt our servers to send your app `GET` requests at predictable endpoints. Each of these requests will return a JSON web token, or _JWT_, as the value of the `signed_payload_jwt` parameter. This article is both a reference and a guide for developing callbacks to handle these events and authenticate our `GET` requests to these endpoints.

```